libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/milestone/VoiceInstructionMilestoneTest.java

```diff
@@ -1,20 +1,17 @@
 package com.mapbox.services.android.navigation.v5.milestone;
 
-import com.mapbox.api.directions.v5.models.LegStep;
-import com.mapbox.api.directions.v5.models.VoiceInstructions;
-import com.mapbox.services.android.navigation.v5.BaseTest;
+import com.mapbox.api.directions.v5.models.DirectionsRoute;
 import com.mapbox.services.android.navigation.v5.routeprogress.RouteProgress;
 
 import org.junit.Test;
 
-import java.util.List;
-
 import static junit.framework.Assert.assertEquals;
 import static junit.framework.Assert.assertFalse;
 import static junit.framework.Assert.assertNotNull;
-import static junit.framework.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
-public class VoiceInstructionMilestoneTest extends BaseTest {
+public class VoiceInstructionMilestoneTest {
```

Comment:
Is this supposed to remove the `extends BaseTest`?

Reply:
Nah, I don't think so because `BaseTest` is still used in other test classes (at least for now). Correct me @danesfeder if I'm wrong.

Reply:
@Guardiola31337 yeah that's correct